cms/djangoapps/contentstore/tests/test_proctoring.py

```diff
@@ -74,6 +75,10 @@ def test_publishing_exam(self, is_time_limited, default_time_limit_minutes,
         a proctored exam, it will also put an entry into the exam tables
         """
 
+        hide_after_due = False
+        if not is_proctored_exam and not is_practice_exam:
```

Comment:
@efischer19 and I talked-- the proctored and practice proctored exams implement their own views. Therefore the hide_after_due property is currently not respected for those exam types.

I think for proctored and practice proctored we just shouldn't pass a value of hide_after_due.